addons/website_sale_mass_mailing/tests/test_is_newletter_enabled.py

```diff
@@ -0,0 +1,61 @@
+from odoo.tests.common import TransactionCase, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteDependentIsEnabled(TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        Website = cls.env['website']
+        MailingList = cls.env['mailing.list']
+
+        cls.mailing_list = MailingList.search([], limit=1)
+        assert cls.mailing_list, "No mailing list found for the test."
+
+        cls.site_one = Website.create({'name': 'Site 1'})
+        cls.site_two = Website.create({'name': 'Site 2'})
+
+        cls.site_one_settings = cls._create_settings_for_website(cls.site_one, enable_newsletter=False)
+        cls.site_two_settings = cls._create_settings_for_website(cls.site_two, enable_newsletter=False)
```

Comment:
The argument is positional.

```suggestion
        cls.site_one_settings = cls._create_settings_for_website(cls.site_one, False)
        cls.site_two_settings = cls._create_settings_for_website(cls.site_two, False)
```